From acd263e47d1f4a0cecdfd1d96573ff8314b43354 Mon Sep 17 00:00:00 2001
From: Arne Morten Kvarving <arne.morten.kvarving@sintef.no>
Date: Thu, 3 Nov 2016 20:17:45 +0100
Subject: [PATCH] no naughty gnomes

---
 client_generic/Makefile.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/client_generic/Makefile.am b/client_generic/Makefile.am
index d996636..d327a95 100644
--- a/client_generic/Makefile.am
+++ b/client_generic/Makefile.am
@@ -51,7 +51,7 @@ installdirs:
 	mkdir -p $(DESTDIR)$(sharedir)/applications
 	mkdir -p $(DESTDIR)$(sharedir)/pixmaps
 	mkdir -p $(DESTDIR)$(sharedir)/electricsheep/icons
-	mkdir -p $(DESTDIR)$(gnome_screensaver_dir)
+#	mkdir -p $(DESTDIR)$(gnome_screensaver_dir)
 
 install-data-local:
 	mkdir -p $(DESTDIR)$(bindir)
@@ -59,7 +59,7 @@ install-data-local:
 	mkdir -p $(DESTDIR)$(sharedir)/applications
 	mkdir -p $(DESTDIR)$(sharedir)/pixmaps
 	mkdir -p $(DESTDIR)$(sharedir)/electricsheep/icons
-	mkdir -p $(DESTDIR)$(gnome_screensaver_dir)
+#	mkdir -p $(DESTDIR)$(gnome_screensaver_dir)
 	cp -rf $(srcdir)/Runtime/Scripts $(DESTDIR)$(sharedir)/electricsheep
 	mkdir -p $(DESTDIR)$(sharedir)/applications/
 	cp -f $(srcdir)/menu-entries/ElectricSheep.desktop.kde $(DESTDIR)$(sharedir)/applications/ElectricSheep.desktop
-- 
2.7.4

